inngest-spring-boot-adapter/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    `java-library`
+    id("io.spring.dependency-management") version "1.1.4"
+}
+
+group = "com.inngest"
+version = "0.0.1-SNAPSHOT"
```

Comment:
probably best to version all the subpackages of `inngest-kt` together. I think there's something we can add in the root gradle file. I'll take a note of this for now